samples/embed.py

```diff
@@ -14,30 +14,30 @@
 # limitations under the License.
 from absl.testing import absltest
 
-import google
 import google.generativeai as genai
-import pathlib
+
 
 class UnitTests(absltest.TestCase):
     def test_embed_content(self):
         # [START embed_content]
         text = "Hello World!"
-        result = genai.embed_content(model="models/text-embedding-004",
-                                     content=text)
-        print(result['embedding'])
+        result = genai.embed_content(model="models/text-embedding-004", content=text)
+        print(result["embedding"])
```

Comment:
Should we use output_depth (or whatever that arg is called) so the print out isn't overwhealming?

Reply:
We can use output dimensionality here. I'll set it here to 10 if that's okay, that way it won't be overwhelmingly long